source/core/security-users.txt

```diff
@@ -145,6 +145,14 @@ the first user in the ``admin`` database. The first user must have
 privileges to create other users, such as a user with the
 :authrole:`userAdmin` or :authrole:`userAdminAnyDatabase` role.
 
+.. versionchanged:: 3.4
+
+   The localhost exception was extended to include the execution of the
```

Comment:
We try to avoid passive voice when there's a straightforward active voice form. In this case, you would probably want `MongoDB 3.4 extended the localhost exception to...`